rotkehlchen/chain/bitcoin_cash/__init__.py

```diff
@@ -0,0 +1,47 @@
+from typing import Dict, List
```

Comment:
I want to suggest a different file structure here.

Something like

```
chain/bitcoin/
|
|--- /btc/
         |
         | --- /utils.py
         |
         ...
|
|---- /bch/
          |
          |
          | ---- /utils.py
          |
         ...
```

This way you can have all common bitcoin functionality under `chain/bitcoin/` and any btc specific stuff to `chain/bitcoin/btc`, any bitcoin cash specific stuff to `chain/bitcoin/bch` and so on and so forth.

Does this make sense?